src/styles/_services.scss

```diff
@@ -2,7 +2,119 @@
   #hero {
     height: 100vh;
     min-height: 700px;
-    background-image: url("/assets/img/services/finsecptx-service-banner-2000.jpg");
+    background-image: url("/assets/img/services/hero-phone-boot-bg-2000.jpg");
+    background-position: center;
+    background-size: cover;
+    div {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      h1 {
+        font-size: 40px;
+        font-weight: 800;
```

Comment:
Can we use the variable `$bold`?

In case we later change the font back to Gothic, then 800 might not be bold, instead it might be 700. Instead of having to change it then all over the shop, we can change it one place only: `app/styles/_variables.scss`.